easystacks/software.eessi.io/2023.06/a64fx/eessi-2023.06-eb-5.0.0-2023a.yml

```diff
@@ -0,0 +1,8 @@
+easyconfigs:
+  - Siesta-5.2.2-foss-2023a.eb:
+      options:
+        # Siesta testsuite fails in Deucalion's a64fx EESSI build environment
+        # but skipping the tests during build and then running them with the 
+        # installed module works, so skipping the tests only in this case
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22799
+        from-commit: 877ac866c1a52576d6f1d44936243ebba908fa7d
```

Comment:
I'm not such a fan of this, this is now linking to a PR that we never plan to merge. Instead, is it possible to override the `buildopts` with a `try-amend` for `buildopts=""`?

Reply:
> I'm not such a fan of this, this is now linking to a PR that we never plan to merge. Instead, is it possible to override the `buildopts` with a `try-amend` for `buildopts=""`?

Good point. Is this what you mean?

```yml
easyconfigs:
  - Siesta-5.2.2-foss-2023a.eb:
      options:
        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22731
        from-commit: b5e36fe564794234cf2d79621ad2d61c3980e5a7
        # Siesta testsuite fails in Deucalion's a64fx EESSI build environment
        # but skipping the tests during build and then running them with the
        # installed module works, so skipping the tests only in this case
        try-ammend: buildopts=""
```

Reply:
Yes (minus the typo `ammend`). I'm not sure it will work, I will give it a quick test

Reply:
Ends up as
```
buildopts = '""'
```
which I guess is fine